### DISCUSSION TITLE
Is it possible to set flag variable to extra features?

### DISCUSSION THREAD

**Discussion by satomi999** [google/lightweight_mmm#72] [Q&A] [answered]
Dear Team :)
Is it possible to add a flag variable for event status (1: event date, 0: not event date) to extra_features?
We know that event increase sales and we want to include event effect when decomposition effects in MMM.

**Comment by pabloduque0** [comment, marked as answer]
Hello @satomi999 !!

Yess! that is definitely a perfect way to add extra_features into the model, binary ones are valid as you mentioned.

Hope it helps!

**Reply by satomi999** [reply]
@pabloduque0 
Thanks for your prompt reply!
Sorry, just one more question.
Regarding multicollinearity, [Robyn](https://facebookexperimental.github.io/Robyn/) uses ridge regression to avoid multicollinearity.
Does LightweightMMM also avoid multicollinearity among variables?

**Reply by pabloduque0** [reply]
Yes, Bayesian methods can be more robust to multi collinearity than ridge regression. You can read more about it [on this paper](https://www.researchgate.net/publication/335063766_A_Bayesian_approach_on_multicollinearity_problem_with_an_Informative_Prior).
